pom.xml

```diff
@@ -584,41 +613,44 @@
         <guava.version.range>[18.0,19.0)</guava.version.range>
         <gson.version>2.6.2</gson.version>
         <gson.version.range>[2.6.2,3)</gson.version.range>
-        <orbit.org.yaml.version>1.16.0.wso2v1</orbit.org.yaml.version>
-        <orbit.org.yaml.version.range>[1.16.0, 1.17.0)</orbit.org.yaml.version.range>
+        <org.snakeyaml.version>1.17</org.snakeyaml.version>
+        <orbit.org.yaml.version.range>[1.17.0,2.0.0)</orbit.org.yaml.version.range>
         <org.objectweb.asm.version>5.1</org.objectweb.asm.version>
         <org.objectweb.asm.version.range>[5.1, 6.0)</org.objectweb.asm.version.range>
         <javax.ws.rs.version.range>[2.0.0, 3.0.0)</javax.ws.rs.version.range>
 
         <!-- UUF Product -->
         <carbon.datasources.version>1.0.0</carbon.datasources.version>
-        <carbon.metrics.version>2.0.0</carbon.metrics.version>
-        <carbon.messaging.version>1.0.11</carbon.messaging.version>
-        <carbon.messaging.version.range>[1.0.11, 2.0.0)</carbon.messaging.version.range>
-        <carbon.transport.http.netty.version>2.1.4</carbon.transport.http.netty.version>
+        <carbon.metrics.version>2.0.1</carbon.metrics.version>
+        <carbon.messaging.version>2.3.1</carbon.messaging.version>
+        <carbon.messaging.version.range>[2.0.0, 3.0.0)</carbon.messaging.version.range>
+        <carbon.transport.http.netty.version>4.3.0-m2</carbon.transport.http.netty.version>
 
         <!-- Handlebars -->
         <orbit.com.github.jknack.handlebars.version>4.0.3.wso2v1</orbit.com.github.jknack.handlebars.version>
         <orbit.com.github.jknack.handlebars.version.range>[4.0.3, 5.0.0)</orbit.com.github.jknack.handlebars.version.range>
         <antlr-version>4.5.1-1</antlr-version>
 
         <!-- MSF4J -->
-        <msf4j.version>2.1.0</msf4j.version>
+        <msf4j.version>2.3.0-m1</msf4j.version>
         <msf4j.version.range>[2.1.0, 3.0.0)</msf4j.version.range>
 
         <!-- Authentication and Authorization -->
-        <carbon.security.caas.version>1.0.0-m2</carbon.security.caas.version>
-        <carbon.security.userstore.jdbc.version>1.0.0-m2</carbon.security.userstore.jdbc.version>
+        <carbon.security.caas.version>1.0.2-m1</carbon.security.caas.version>
+        <carbon.security.caas.version.range>[1.0.0, 2.0.0)</carbon.security.caas.version.range>
+        <!--carbon.security.userstore.jdbc.version>0.1.3</carbon.security.userstore.jdbc.version-->
```

Comment:
If this property is no longer used, then let's remove it rather than commenting-out.